### DISCUSSION TITLE
[mini.pick] How do I access the item from a custom action?

### DISCUSSION THREAD

**Discussion by ndavd** [nvim-mini/mini.nvim#629] [Q&A] [answered]
Trying to create a custom mapping which needs the item data. Kind of like it was a secondary `choose` operation.

**Comment by echasnovski** [comment, marked as answer]
The [`MiniPick.get_picker_matches().current`](https://github.com/echasnovski/mini.nvim/blob/ea1af8c7d5e72148cae8a04e9887322a53fe66cf/doc/mini-pick.txt#L1214) should do the trick.

**Reply by ndavd** [reply]
Thanks, that works
